resources/build/version/src/reportHistory.ts

```diff
@@ -0,0 +1,136 @@
+import { warning as logWarning } from '@actions/core';
+
+import { GitHub } from '@actions/github';
+
+import { findLastHistoryPR, getAssociatedPR} from './graphql/queries';
+import { spawnChild } from './util/spawnAwait';
```

Comment:
I'm surprised that `reportHistory` here doesn't seem to reuse a lot of `history`'s code.  I'd figure that `reportHistory` is a simple variation on `history`:  "report" vs "report, then write".

Reply:
Don't worry, it reuses a lot of `version`'s code -- as you can see in the source here!

Reply:
The reason `version` was written rather than using `history.sh` was there was heavy json lifting and web API calls required which bash really does not do well with, and this ended up being cleaner.